src/components/stockDiscoveryComponents/tickercard/Tickercard.js

```diff
@@ -6,8 +6,6 @@ import { Card, ListGroupItem, ListGroup, Row, Col, Container } from 'react-boots
 
 import { Link } from "react-router-dom";
 import SentimentBadge from '../../widgets/sentimentBadge/SentimentBadge';
-import InfoButtonHover from "../../widgets/InfoButtonHover/InfoButtonHover"
```

Comment:
Just double checking this was meant to be removed?

Reply:
Commenting here for posterity, @beckskelly replied to me on Discord about this. It was indeed supposed to be removed